### ISSUE TITLE
Issue Template: Make every issue actionable

### ISSUE THREAD

**Issue by mekarpeles** [internetarchive/openlibrary#9693]
### TL;DR

Update issue template for bugs and features to add `Needs: Breakdown` as default label upon creation (and make sure both issue templates have a section for leads to add e.g. files, accepted solution, designs, checklist/acceptance criteria).

### Problem we're solving

Today, it's easy to tell which issues don't have `leads` and haven't been `triaged`, but it's quite hard to know which issues are actionable, i.e. have `breakdown` (this means: accepted solutions/approach, design, acceptance criteria, checklist, etc).

### Background

When a new issue is created, a few things are important:
1. The issue must be created with `detail` about the issue, its `impact`, reproducibility/evidence (everything needed to evaluate and assign) 
2. Someone with expertise and accountability needs to be assigned as the `lead` so the community can ask questions
3. This `lead` needs to `triage` and prioritize the issue
4. This issue needs to have an `accepted solution` by the `lead` (which may be informed by a contributor)
5. Once there is an `accepted solution` the `lead` or `assignee` should provide a `breakdown`

### Proposed Solution

Once there is an issue with `detail`, and a `lead` is appointed, and the issue has been `triaged`, a `accepted solution` must be provided with a `breakdown` before it is ready to be `assigned` and completed.

When a new `bug` is created, the issue template will automatically add a new tag `Needs Breakdown` which we will define as: 

* accepted solutions/approach
* designs (if applicable)
* checklist of steps / acceptance criteria

### Reservations

Some staff members are concerned it may require a lot of work to break down every issue, list files, have designs, etc, and may only be necessary in certain cases, e.g. where we expect an issue to be done by a member of the community.

In these cases, we give the lead discretion to simply remove the automatically added `Needs: Breakdown` label.

**Comment by mekarpeles** [comment]
Closed by #9771
